test/while_test.js

```diff
@@ -78,6 +78,14 @@ describe('while', () => {
     `);
   });
 
+  it('turns `loop` with an inline body into an inline `while` loop with an always-true condition', () => {
+    check(`
+      loop a
```

Comment:
We actually [already have one](https://github.com/decaffeinate/decaffeinate/blob/1e38f68f727674065fe15a9e8b88f8fa518b3cb7/test/while_test.js#L208-L214).